src/get-started/new-typography-scale/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: layout-pane.njk
+title: Updating your service to use the new typography scale
+section: Get started
+theme: How to guides
+order: 6
+description: The GOV.UK Design System team have been working on updating our typography scale provided with GOV.UK Frontend. This new scale is now available to use and will be enforced by default in GOV.UK Frontend version 6.0.0. Here's how to use it, what's changed and what to expect when implementing it into your service.
+---
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+The GOV.UK Design System team have been working on updating our typography scale provided with GOV.UK Frontend. This new scale is now available to use and will be enforced by default in GOV.UK Frontend version 6.0.0. Here's how to use it, what's changed and what to expect when implementing it into your service.
+
+## Changes we've made
+The new typography scale includes the following changes from the previous scale:
+
+{{ govukTable({
+    caption: "Changes to the typography scale",
+    captionClasses: "govuk-table__caption--m",
+    head: [
+        {
+            text: "Typography scale value"
+        },
+        {
+            text: "Change"
+        }
+    ],
+    rows: [
+        [
+            {
+                text: "14"
+            },
+            {
+                text: "Deprecated as a scale value and will be removed in GOV.UK Frontend version 6.0.0"
+            }
+        ],
+        [
+            {
+                text: "16"
+            },
+            {
+                text: "16px across all screen sizes and no longer drops to 14px on small screens"
+            }
+        ],
+        [
+            {
+                text: "19"
+            },
+            {
+                text: "19px across all screen sizes and no longer drops to 16px on small screens"
+            }
+        ],
+        [
+            {
+                text: "24"
+            },
+            {
+                text: "21px instead of 18px on small screens, plus line height adjustment"
```

Comment:
Can I check my understanding – 24px used to drop to 18px, but now it's only dropping to 21px? If so, we might want to amend the text a little to clarify unless it's more important to stay consistent with the other text entries.